src/types/emr/encounter.ts

```
@@ -1,7 +1,8 @@
 import { Patient } from "@/types/emr/newPatient";
+import { FacilityOrganization } from "@/types/facilityOrganization/facilityOrganization";
 import { UserBase } from "@/types/user/user";
 
 import { FacilityOrganization } from "../facilityOrganization/facilityOrganization";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'FacilityOrganization'. Consider to delete it or rename it.

'FacilityOrganization' is defined here:


(lint/suspicious/noRedeclare)